src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }: Props) => (
+  <article
+    data-id={todo.id}
+    className={`TodoInfo${todo.completed ? ' TodoInfo--completed' : ''}`}
+  >
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+
+    {todo.user
+      ? <UserInfo user={todo.user} />
+      : <></>}
```

Comment:
isn't this what you're trying to do here?
```suggestion
    {todo.user && (
      <UserInfo user={todo.user} />
    )}
```